blog/electron-22-0.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Electron 22.0.0
+date: 2022-11-29T00:00:00.000Z
+authors:
+    - name: VerteDinde
+      url: 'https://github.com/VerteDinde'
+      image_url: 'https://github.com/vertedinde.png?size=96'
+    - name: georgexu99
+      url: 'https://github.com/georgexu99'
+      image_url: 'https://github.com/georgexu99.png?size=96'
+slug: electron-22-0
+
+---
+
+Electron 22.0.0 has been released! It includes upgrades to Chromium `108`, V8 `10.8`, and Node.js `16.17.1`. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 22.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on Twitter, or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Stack Changes
+
+* Chromium `108`
+    * [New in Chrome 108](https://developer.chrome.com/blog/new-in-chrome-108/)
+    * [New in Chrome 107](https://developer.chrome.com/blog/new-in-chrome-107/)
+    * [New in DevTools 108](https://developer.chrome.com/blog/new-in-devtools-108/)
+    * [New in DevTools 107](https://developer.chrome.com/blog/new-in-devtools-107/)
+* Node.js `16.17.1`
+    * [Node 16.17.1 blog post](https://nodejs.org/en/blog/release/v16.17.1/)
+* V8 `10.8`
+
+### Highlighted Features
+
+### UtilityProcess API [#36089](https://github.com/electron/electron/pull/36089)
+
+The new `UtilityProcess` main process module allows the creation of a lightweight Chromium child process with only Node.js integration while also allowing communication with a sandboxed renderer using `MessageChannel`. The API was designed based on Node.js `child_process.fork` to allow for easier transition, with one primary difference being that the entry point `modulePath` must be from within the packaged application to allow only for trusted scripts to be loaded, unlike the Node.js version which can load from an arbitrary location.
+
+Additionally the module prevents establishing communication channels with renderers by default, upholding the contract in which the main process is the only trusted process in the application. Instead the main process is responsible for creating the desired communication channel and sharing the endpoints to the respective child processes.
+
+```typescript title='main.js'
```

Comment:
IMO, I think adding the sample code might be too much here - the utility process API is a neat feature, but it's taking up a lot of space in the blog post. Could we include this sample code in the docs themselves and link to the utility process docs?